AGENTS.md

```
@@ -10,39 +10,42 @@ tp2 is a full-stack TypeScript application built with:
 - **Styling**: Tailwind CSS v4
 - **UI Components**: shadcn/ui with Base UI and Radix UI
-- **Authentication**: Clerk
+- **Authentication**: `@convex-dev/auth` (Password + Anonymous providers)
 - **State/Data Fetching**: TanStack Query integrated with Convex
 - **Testing**: Vitest with Testing Library
-- **Linting/Formatting**: Biome
+- **Linting/Formatting**: Biome (primary) + ESLint (specialty plugins only)
 
 ## Project Structure
 
 ```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)